guest/api/types.go

```diff
@@ -91,9 +91,20 @@ type ScorePlugin interface {
 	Score(state CycleState, pod proto.Pod, nodeName string) (int32, *Status)
 }
 
+// ScoreExtensions is a WebAssembly implementation of framework.ScoreExtensions.
+type ScoreExtensions interface {
+	Plugin
+
+	NormalizeScore(state CycleState, pod proto.Pod, scores NodeScoreList) (map[string]int, *Status)
+}
+
 type NodeInfo interface {
 	// Metadata is a convenience that triggers Get.
 	proto.Metadata
 
 	Node() proto.Node
 }
+
+type NodeScoreList interface {
+	NodeScoreList() map[string]int
```

Comment:
It's also a similar one, let's change like:

```suggestion
type NodeScore interface {
	// Map returns a map
	// which is keyed by the node name and valued by the score.
	Map() map[string]int
```

Reply:
It's a general suggestion, when you use map in interface, it's always better to say explicitly what is key and what is value.

Reply:
Thanks, fixed it